solaris/solaris10/generate_wazuh_packages.sh

```diff
@@ -121,9 +121,9 @@ installation(){
     arch="$(uname -p)"
     # Build the binaries
     if [ "$arch" == "sparc" ]; then
-        gmake -j $THREADS TARGET=agent PREFIX=$INSTALL USE_SELINUX=no USE_BIG_ENDIAN=yes
+        gmake -j $THREADS TARGET=agent PREFIX=$INSTALL USE_SELINUX=no USE_BIG_ENDIAN=yes DISABLE_SHARED=no
```

Comment:
The `DISABLE_SHARED` flag must be set to `no` in order to build the static library instead of the shared object.